src/sourcekit-lsp/LanguageClientManager.ts

```diff
@@ -527,6 +533,35 @@ export class LanguageClientManager {
             this.workspaceContext.outputChannel.log(`SourceKit-LSP setup`);
         }
 
+        client.onNotification(langclient.LogMessageNotification.type, params => {
+            const sourceKitParams = params as SourceKitLogMessageParams;
+            if (sourceKitParams.logName) {
+                const outputChannel =
```

Comment:
Can we create the log channels in advance. We should know what logs SourceKit-LSP is going to want to open.

Reply:
After a chat will Alex, he asked if we could show timestamp but not log level for the Indexing log because it will always be info. So, for known logs like that, I'm letting set that configuration on startup. My idea behind allowing unknown logs to be created is if sourcekit-LSP changes, for development or as a user, they don't need to wait for an extension release to see the new log. Let me know if you still have concerns